product-service/src/handlers/getProductsById.ts

```diff
@@ -0,0 +1,31 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+import { productsList } from '../mockProductsList';
+
+export const getProductsById: APIGatewayProxyHandler = async event => {
+  try {
+    const { productId } = event.pathParameters;
+
+    const productById: Product = productsList.find(product => product.id === productId);
+
+    return productById 
+        ? {
+            statusCode: 200,
+            headers: { 
+              'Access-Control-Allow-Origin': '*',
+              'Access-Control-Allow-Credentials': true 
+            },
+            body: JSON.stringify(productById)
+          } 
+          : {
+            statusCode: 404,
+            body: JSON.stringify({ error: `Product with id: ${productById} is not found` })

```

Comment:
Snth wrong here https://tytd8guixe.execute-api.eu-west-1.amazonaws.com/dev/products/7567ec4b-b10c-48c5-

Reply:
Just a typo.Change `productById` to `productId`

Reply:
@helly-fox Thanks for noticing this, fixed